base/fs.cpp

```diff
@@ -35,6 +39,46 @@ const std::string::value_type* path_separators = "\\/";
 const std::string::value_type* path_separators = "/";
 #endif
 
+#if LAF_MACOS
+static bool* g_isfilesystemCaseSensitive = nullptr;
+
+bool isFilesystemCaseSensitive() {
+  if (g_isfilesystemCaseSensitive)
+    return *g_isfilesystemCaseSensitive;
+  std::string path1 = get_current_path() + "/filesystemTest.txt";
+  std::string path2 = get_current_path() + "/FilesystemTest.TXT";
+
+  std::ofstream file1(path1);
```

Comment:
This can fail in a lot of different ways (e.g. we don't have access to write in the current path). I'd try to avoid checking this on macOS, or investigate the correct way to check if the file system is case sensitive (it might vary from file system/mount point, so I'm not sure if I'd worry about this).

Reply:
Ok. In conclusion, should we keep the macOS case-sensitive format as it was originally? Or adopt the new case-insensitive file name comparison, as described in the first pull request?

Reply:
I think something like https://github.com/aseprite/laf/commit/3651931d55c4b9380c3a2004900e97829c5c611a is enough, Linux being the only platform where case-sensitive is allowed, and Windows/macOS case-insensitive.